README.md

```diff
@@ -10,7 +10,16 @@ Returns fiat prices for base currencies and ERC20 tokens.
 
 ## Configuration
 
-Environment variables:
+```bash
+cp .env.sample .env
+# You can configure variables in .env
+docker compose build
+docker compose up
+```
+
+Then go to http://localhost:8000
```

Comment:
I'm thinking that we are mixing configuration and execution here, could we separate them in different sections?

Reply:
You are right